src/org/usfirst/frc/team1675/robot/Robot.java

```diff
@@ -50,7 +50,7 @@
 		try{
 			driveBase = new DriveBase();
 			clawSub = new ClawSubSystem(false); //false everywhere
-			clawArm = new ClawArm(true); // true for practice robot, false for competition robot
+			clawArm = new ClawArm(false); // true for practice robot, false for competition robot
```

Comment:
Yes @aMueller1030 tested it and confirmed limit switches worked correctly. It looks like that change was made at Rockford this summer as it was already on the laptop they worked on. My guess is we made that same change at champs as well but just didn't commit it.